ssl/packet.c

```diff
@@ -412,6 +412,12 @@ unsigned char *WPACKET_get_curr(WPACKET *pkt)
     return GETBUF(pkt) + pkt->curr;
 }
 
+void WPACKET_advance_write(WPACKET *pkt, size_t len)
+{
+    pkt->written += len;
```

Comment:
Does this need to do a bounds check?

Reply:
Maybe? I don't believe the WPACKET APIs are public, so it's only used internally. As used herein, the WPACKET buffer is grown to account. Perhaps an additional comment indicating use?

Reply:
I think one of the main goals of this API is to make sure no buffer overwrites happen. You are probably better to do the check so it doesnt bite someone in the future.